src/components/PostInfo/PostInfo.tsx

```diff
@@ -0,0 +1,37 @@
+import React from 'react';
+import UserInfo from '../UserInfo/UserInfo';
+import CommentInfo from '../CommentInfo/CommentInfo';
+import { PreparedPost } from '../../types/PreparedPost';
+import './PostInfo.scss';
+
+type Props = {
+  post: PreparedPost,
+};
+
+const PostInfo: React.FC<Props> = ({ post }) => {
+  const {
+    title,
+    body,
+    user,
+    writtenComments,
+  } = post;
+
+  return (
+    <div className="post">
+      <h2 className="post__title">
+        {title}
+      </h2>
+      <p className="post__body">
+        {body}
+      </p>
+      {user ? (
+        <UserInfo user={user} />
+      ) : (
+        <p>Someone</p>
+      )}
```

Comment:
maybe better use && instead ternar, and if user will be null, we can show nothing)

Reply:
As a variant

Reply:
Really good point! Changed it 👌